docker/docker-compose.yml

```diff
@@ -0,0 +1,72 @@
+version: '3.1'
+services:
+  create-metadata-bucket:
```

Comment:
so cool :)

Reply:
hacky but works.. and will stop once finished